Fix: Glossary for Cloud (#36026)
mdn/content: files/en-us/glossary/cloud/index.md

---
title: Cloud
slug: Glossary/Cloud
page-type: glossary-definition
---

{{GlossarySidebar}}

> **Cloud** or **Cloud Computing** refers to the on-demand delivery of a wide range of computing services such as servers, storage, databases, networking, software, analytics, and intelligence via the Internet.

Reviewer: You could also consider mentioning data centers, and provide some examples of some of the services you mention above.

I'd remove "software" from your list. Most if not all of them are software, so it seems to be a bit too vague.

> **Cloud** or **Cloud Computing** refers to the on-demand delivery of a wide range of computing services such as servers, storage, databases, networking, software, analytics, and intelligence via the Internet.

Reviewer: Also, when you say "intelligence", what do you mean? AI? Again, I'd remove that, unless you want to instead provide examples of tools that use AI or LLMs.

> This model enables faster innovation, flexible scalability, and significant cost savings. Users can access and utilize these services through a _pay-as-you-go_ pricing model, ensuring they only pay for what they use, making it a highly efficient and adaptable solution for businesses and individuals alike.

Reviewer: Mention "Software as a service" here, and its relationship with cloud pay-as-you-go and subscription models?

## See also

> - [What is Cloud Computing?](https://cloud.google.com/learn/what-is-cloud-computing?hl=en) on Google Cloud

Reviewer: Also link to https://en.wikipedia.org/wiki/Cloud_computing?

---
title: Cloud
slug: Glossary/Cloud
page-type: glossary-definition
---

{{GlossarySidebar}}

> **Cloud** refers to servers that are accessed over the Internet, and the software and databases that run on those servers.

Reviewer: ```suggestion
The **Cloud** refers to servers located worldwide that are accessed over the Internet, and the software and databases that run on those servers.
```

> Instead of storing data or running programs on a local computer or server, users rely on these remote resources. Cloud servers are located in data centers all over the world. By using {{glossary("Cloud Computing")}}, users and companies do not have to manage physical servers themselves or run software applications on their own machines.

Reviewer: ```suggestion
Instead of storing data or running programs on a local computer or server, users can access cloud resources to perform tasks: this is known as {{glossary("Cloud computing")}}.
```

> ### Key features of the Cloud

Reviewer: ```suggestion
### Key features of the cloud
```

---
title: Cloud
slug: Glossary/Cloud
page-type: glossary-definition
---

{{GlossarySidebar}}

The **Cloud** refers to servers located worldwide that are accessed over the Internet, and the software and databases that run on those servers.

Instead of storing data or running programs on a local computer or server, users can access cloud resources to perform tasks: this is known as {{glossary("Cloud computing")}}.

> ### Key features of the cloud

Reviewer 2: ``` suggestion
Key features of the cloud include:
```
